spec/features/quotes_spec.rb

```diff
@@ -18,4 +18,17 @@
     expect(page).to have_content("Something cool")
   end
 
+  scenario 'Users can see a quote creation date' do
+    create_user email: "user@example.com"
+    Quote.create!(text: %Q{Something pithy}, author: %Q{Someone}, created_at: (Date.today-6))
+
+    visit root_path
+    click_on "Login"
+    fill_in "Email", with: "user@example.com"
+    fill_in "Password", with: "password"
+    click_on "Login"
+
+    expect(page).to have_content("7 days ago")
+  end
```

Comment:
This is a good test - visits the page and lets us know if the time_in_words function behaves as expected, but can you also update the quote record so that null dates are not allowed? I know you're new to rails so you should look into.

http://guides.rubyonrails.org/active_record_validations.html